### ISSUE TITLE
Add question: "Do visual text and screen reader text must absolutely match?"

### ISSUE THREAD

**Issue by MaximPerry** [cenw-wscoe/WCAG-in-Plain-Language#47]
Suggested answer: 
No, as long as they both provide the same general information to users. It becomes an issue when a screen reader text is providing aditional information not being visually communicated, or vice-versa. 

Example of a failure: an aria-label that reads: "Password - must contain at least 8 characters" on a label that just visually reads "Password" (where those additional instructions aren't visually displayed elsewhere either).

**Comment by Jefferydo** [comment]
rephrased question: ""Do visual text and screen reader text have to match?""

"No, as long as they both provide the same general information to users."

I'm pretty sure visible text has to be faithfully communicated to the screen reader user and voice input user.

- Images of text must have alt text that matches. Exception: extraneous or redundant text in the image. 
- Labels of controls must have an ARIA label that matches or contains the visible text label (SC 2.5.3: Label in Name).
- As for the rest, hiding text nodes from screen reader users with aria-hidden is allowed only for extraneous or redundant text, though the spec goes on to say if you do it anyway, you must provide identical or equivalent meaning:
> Authors MAY, with caution, use aria-hidden to hide visibly rendered content from assistive technologies only if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using aria-hidden to hide visible content from screen readers MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies.

Suggested answer:
Yes, unless the text is extraneous or redundant.
